### PR TITLE
Milestone2 of Calculator React Solo Project

### PR DESCRIPTION
**Pull request by carlosprietobarron**
In this milestone : 

Create Display component

Create Button Panel component and button components

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @carlosprietobarron
### You have done well so far :clap: However, you have a few issues to fix:-
- [ ] Add linting files as recommended by Microverse, install all the required dependencies, configure them well and run commands `npx eslint .` and `npx stylelint "**/*.{css,scss}"` If errors kindly solve them. [Follow this guide](https://github.com/microverseinc/linters-config/tree/master/react-redux)
- [ ] We recommend the use of es6 arrow functions rather than the function keyword. See example `const Btngroup = (props) => { # Your code }`
- [ ] The script `npm start` is not working, this is required to start the app in the app. See image below
npm start
- [ ] Remove unwanted files like App.test.js and setupTests.js
- [ ] Files like App.js is to be added in the directory components
### Make all these changes and submit for another code review. Good luck.
- Happy coding!

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @carlosprietobarron  👋

**Good job on this. The project is almost in the clear.**

Just a change to be made.

- Please refer to the comment on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  //"extends": ["airbnb", "plugin:react/recommended"],
```
Comment (mikenath223) [inline .eslintrc.json]: - [ ] Kindly uncomment this. You should use config rules shared by other students. And that line is one of the required rules.

File: src/components/display.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+const Display = props => {
+  const { result } = props;
+
+  return (
+    <div className="calc-display">
+      <span>{result}</span>
+    </div>
+  );
+};
+/* eslint-enable react/prop-types, no-console, arrow-body-style */
```
Comment (mikenath223) [inline src/components/display.js]: - [ ] Please add proptype validation here.

File: src/components/btnPanel.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+      <Btngroup group={group1} />
+      <Btngroup group={group2} />
+      <Btngroup group={group3} />
+      <Btngroup group={group4} />
+      <Btngroup group={group5} />
+    </div>
+  );
+};
+/* eslint-enable react/prop-types, no-console, arrow-body-style */
+export default ButtonPanel;
```
Comment (mikenath223) [inline src/components/btnPanel.js]: - [ ] Kindly also add proptypes validation here

File: src/components/Buttn.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+  return (
+    <div className="div-key">
+      <button type="button" id={name} className="key-btn">{symbol}</button>
+    </div>
+  );
+};
+
+export default Buttn;
+/* eslint-enable react/prop-types, no-console, arrow-body-style */
```
Comment (mikenath223) [inline src/components/Buttn.js]: - [ ] Please add proptypes validation here.

**Comment by mikenath223** [comment]
- I think you made the mistake of requesting a review on milestone1 when you actually might have wanted to request a review on milestone2 branch instead. If this isn't the case then kindly make these changes. But otherwise, make the changes on the correct branch.

**Review by bellom — APPROVED** [review]
## Hi @carlosprietobarron  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

**NOTE:**_Few things you should look into before merging or moving to the next milestone;_

**1.** For some reason you added Node server, please you don't need this.
Please let's carefully read and follow the instructions in the pathwright as we both know all Microverse students are bind to follow the project requirements and instructions.

**2.** Let's be professional with the naming of our feature-branch. Kindly name your branch according to the milestone you're working on.
In your case instead of using `milestone1`, you're working on `milestone2`, you should have named your branch `milestone2`.

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request closed without merging**